rpcs3/rpcs3qt/localized.cpp

```diff
@@ -4,29 +4,24 @@ Localized::Localized()
 {
 }
 
-QString Localized::GetVerboseTimeByMs(qint64 elapsed_ms, bool show_days) const
+QString Localized::GetVerboseTimeByMs(quint64 elapsed_ms, bool show_days) const
 {
-	if (elapsed_ms <= 0)
-	{
-		return tr("Never played");
-	}
-
-	const qint64 elapsed_seconds = (elapsed_ms / 1000) + ((elapsed_ms % 1000) > 0 ? 1 : 0);
-	qint64 hours = elapsed_seconds / 3600;
+	const quint64 elapsed_seconds = (elapsed_ms / 1000) + ((elapsed_ms % 1000) > 0 ? 1 : 0);
```

Comment:
Should this have been '> 499' to round to the nearest second?

Reply:
No